composer.json

```diff
@@ -4,7 +4,8 @@
   "type": "wordpress-plugin",
   "require": {
     "composer/installers": "^1.0",
-    "moxie-lean/wp-endpoint": "*"
+    "moxie-lean/wp-endpoint": "*",
+    "Epoch2/http-codes": "*"
```

Comment:
Can we use specifc version numbers like `^1.0.0` in order to prevent major problems.